getting-started/faqs.md

```diff
@@ -4,220 +4,197 @@
 
 ### About CFT
 
-**1. What is Cloud File Transfer (CFT)?**
+**What is Cloud File Transfer (CFT)?**
 
-A centralized, fully managed file transfer service for the whole of government (WOG). CFT is designed to help you send and receive large files securely across zones. The sender and receiver can be an application or an end-user. Use CFT to transfer files between your intranet and internet zones or exchange files with your business partners and vendors as well as other agencies. 
+A centralized, fully managed file transfer service for the whole of government (WOG).
 
 CFT is a part of the SG Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).
 
-**Key Features:**
+Core capabilities:
 
-**Compliant file transfers**
+- Compliant cross-zone file transfers according to IM8
```

Comment:
//typical biz users might not understand the term cross-zone expand this to illustrate what the diff zones are, can be a new Q too, what do you think?

e.g.
"What is cross-zone file transfer?"